cumulus/scripts/migrate_storage_to_genesis/identityMigration.js

```diff
@@ -0,0 +1,108 @@
+const { u8aToHex, hexToString } = require('@polkadot/util');
+const { xxhashAsHex } = require('@polkadot/util-crypto');
+
+const migrateIdentity = async (key, data, api) => {
+	// Register the new `Registration` format type
+	api.registry.register({
+		IdentityInfoNew: {
+		_fallback: 'IdentityInfoTo198',
```

Comment:
Mainly out of curiosity - what is this fallback and where does the `IdentityInfoTo198` come into it?

Reply:
It is coming from `IdentityInfo` declaration in polkadot-js/types. It is a fallback to a previous `IdentityInfoTo198` definition I guess in case the encoding fails.

I can probably remove it.